src/cs/lib/msquic_generated.cs

```diff
@@ -280,6 +281,9 @@ internal unsafe partial struct QUIC_CREDENTIAL_CONFIG
 
         internal QUIC_ALLOWED_CIPHER_SUITE_FLAGS AllowedCipherSuites;
 
+        [NativeTypeName("const char *")]
```

Comment:
@nibanks How does .NET handle adding a member in the middle of an object?

Reply:
It's at the end.

Reply:
There's all the pointers to the certificate data after it, I don't think this is the end.

Reply:
Look at msquic.h for the actual structure. Everything below this is a property (i.e. just a helper function).